### PR TITLE
aggregation: change for isFiction

### PR DESCRIPTION
**Pull request by rerowep**
* Needs rero-ils:#3616

### CODE REVIEW

File: projects/admin/src/app/service/bucket-name.service.ts [hunk 1]
```diff
@@ -50,9 +50,12 @@ export class BucketNameService implements IBucketNameService {
   transform(aggregationKey: string, value: string): Observable<string> {
     console.log();
     switch (aggregationKey) {
+      case 'fiction':
+        const fiction_label = Boolean(value)? _('Fiction') : _('No fiction');
+        return of(this.translateService.instant(fiction_label, { count: value }));
```
Comment (Garfield-fr) [inline projects/admin/src/app/service/bucket-name.service.ts:55]: Remove `, { count: value }`

**Review by Garfield-fr — CHANGES_REQUESTED** [review]

**Review by Garfield-fr — APPROVED** [review]

File: projects/admin/src/app/service/bucket-name.service.ts [hunk 2]
```diff
@@ -48,11 +48,13 @@ export class BucketNameService implements IBucketNameService {
...
   transform(aggregationKey: string, value: string): Observable<string> {
-    console.log();
     switch (aggregationKey) {
+      case 'fiction':
+        const fiction_label = Boolean(value)? _('Fiction') : _('No fiction');
+        return of(this.translateService.instant(fiction_label));
       case 'claims_count':
-        const label = Number(value) < 2 ? _('{{count}} claim') : _('{{count}} claims');
-        return of(this.translateService.instant(label, { count: value }));
+        const claims_label = Number(value) < 2 ? _('{{count}} claim') : _('{{count}} claims');
```
Comment (jma) [inline projects/admin/src/app/service/bucket-name.service.ts:56]: Check with @Garfield-fr to rewrite this using a simple way.

**Review by jma — APPROVED** [review]

**Review by lauren-d — APPROVED** [review]

File: projects/admin/src/app/routes/documents-route.ts [hunk 3]
```diff
@@ -125,8 +125,8 @@ export class DocumentsRoute extends BaseRoute implements RouteInterface {
               'language',
```
Comment (PascalRepond) [inline projects/admin/src/app/routes/documents-route.ts:125]: I would put:

```typescript
            aggregationsOrder: [
              'document_type',
              'organisation',
              'language',
              'year',
              'author',
              'subject',
              'genreForm',
              'fiction',
              'intendedAudience',
              'acquisition',
              'status',
            ],
            aggregationsExpand: [
              'document_type',
              'organisation',
              'language',
              'year',
              'author',
              'subject',
            ],
```

That function in aggregationsExpand seems useless. @jma what do you think?

**Review by PascalRepond — CHANGES_REQUESTED** [review]

File: projects/admin/src/app/routes/documents-route.ts (hunk 3 above)
Comment (Garfield-fr) [reply projects/admin/src/app/routes/documents-route.ts:125]: @PascalRepond @jma This code has nothing to do with this PR. If it is necessary to correct, a new PR must be created to explain the reasons for the change.

**Review by PascalRepond — APPROVED** [review]

**Pull request merged**